pkg/controllers/managementuser/resourcequota/resource_quota_validate.go

```diff
@@ -57,3 +56,49 @@ func (r *reconcileController) reconcileNamespaces(_ string, p *apiv3.Project) (r
 	}
 	return nil, nil
 }
+
+func isEmpty(rql *apiv3.ResourceQuotaLimit) bool {
+	if rql.ConfigMaps != "" {
```

Comment:
This could this be simplified to `return reflect.ValueOf(rql).IsZero()`

Reply:
Thanks for this